src/Route.php

```diff
@@ -252,6 +276,11 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
     {
         if ($this->stack === null) {
             for ($i = count($this->middlewares) - 1; $i >= 0; $i--) {
```

Comment:
I think would be nice to change the loop from `for` to `foreach`. What do you think?

Reply:
Are you sure that a call of the array_reverse() is a good idea?